docs/src/languages/php/composer-auth.md

```diff
@@ -1,20 +1,25 @@
 ---
 title: "Authenticated Composer repositories"
 sidebarTitle: "Authenticated Composer"
-description: Some PHP projects may need to use a private, third-party Composer repository in addition to the public Packagist.org repository. Often, such third party repositories require authentication to download packages. These credentials shouldn't be located in the Git repository source code for security reasons.
+description: Allow Composer to authenticate against a private third-party Composer repository and download PHP packages from it.
 ---
 
-{{% description %}}
+[Packagist](https://packagist.org/) is the primary Composer repository for public PHP packages.
+But you can also have Composer download PHP packages from a private, third-party Composer repository.
+To make sure Composer has the necessary credentials to do so,
+follow the instructions on this page.
 
-To handle that situation, you can define a `env:COMPOSER_AUTH` [project variable](../../development/variables/set-variables.md#create-project-variables) which allows you to set up authentication as an environment variable.
-The contents of the variable should be a JSON formatted object containing an `http-basic` object (see [composer-auth specifications](https://getcomposer.org/doc/03-cli.md#composer-auth)).
+## Before you begin
 
-The advantage is that you can control who in your team has access to those variables.
+You need:
+- A Platform.sh project using [WordPress with Composer](../../guides/wordpress/composer/_index.md)
```

Comment:
:question: Does it have to be WordPress? It seems that PHP uses for [Composer for all dependencies](https://auth-composer-7avagwi-652soceglkw4u.eu-3.platformsh.site/languages/php.html#dependencies). I'd think it would be any running PHP project that doesn't use a different dependency manager.

Reply:
Ooh, that's a very good point!